packages/esm-patient-labs-app/src/routes.json

```diff
@@ -35,6 +35,10 @@
       "component": "labOrderPanel",
       "slot": "order-basket-slot",
       "order": 2
+    },
+    {
+      "name": "results-viewer-dialog",
+      "component": "resultsViewerDialog"
     }
```

Comment:
This should be registered as a [modal](https://github.com/openmrs/openmrs-esm-core/pull/961), not an extension. Additionally, the modal file name ought to match the [naming convention](https://github.com/openmrs/openmrs-esm-patient-chart/pull/1875).

Reply:
https://o3-docs.openmrs.org/docs/coding-conventions#naming